signer-service/src/api/controllers/moonbeam.controller.js

```diff
@@ -46,3 +50,26 @@ exports.executeXcmController = async (req, res) => {
     res.status(500).send({ error: 'Internal Server Error' });
   }
 };
+
+exports.sendStatusWithPk = async () => {
+  const moonbeamExecutorAccount = privateKeyToAccount(MOONBEAM_EXECUTOR_PRIVATE_KEY);
```

Comment:
Let's move this statement into the try{} block below as this can throw an error if the moonbeam private key is undefined or invalid.

Reply:
If you want to make it available in the catch block as well, we would need to make it 
```
let moonbeamExecutorAccount;

try {
  moonbeamExecutorAccount = ...
```

and change the catch to
```
    return { status: false, public: moonbeamExecutorAccount?.address };
```

Reply:
True. I was thinking that if it is undefined the service will not even start (we have that check), but it very well could be invalid.
I'll go for the second suggestion such that is easier to debug in that case.

Reply:
Actually, it is more or less the same since if it fails it will not be assigned anyway. So a `public = undefined` will hint that the key is invalid.

Reply:
Thanks 👍  we still need that 
```
    return { status: false, public: moonbeamExecutorAccount?.address };
```
otherwise it might throw an error because it cannot find `address` on `undefined`.

Reply:
Of course!